cdap-ui/app/cdap/components/Market/AllTab/AllTabContents.scss

```diff
@@ -24,7 +26,27 @@
       overflow-y: auto;
 
       .tab-pane { width: 100%; }
+
       .all-tab-content {
+        .search-box {
+          margin-top: 10px;
+          width: 100%;
+          padding-left: 15px;
+          padding-right: 35px;
+
+          .input-group-text {
+            background-color: #ffffff;
```

Comment:
Please use already existing color. The closest color to this would be `$grey-08`.

Reply:
done